iOS/README.md

```diff
@@ -0,0 +1,42 @@
+# PlayFab Multiplayer SDK for iOS
+
+## Logging

```

Comment:
rather than adding new files wouldn't it be simpler to just add these new logging instructions to the logging README that already exists under `TraceScripts/`?

Reply:
I agree we could move the content of READMEs from under iOS/macOS folders into the common README under TraceScripts